Makefile.in

```diff
@@ -318,13 +318,20 @@ distprep: catman-do
 	$(AUTORECONF)
 	-rm -rf autom4te.cache
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-iron-files install-sysconf host-key check-config
```

Comment:
Probably just `ironsftp`, but I was trying to avoid conflicts in future upstream merges. That said, it would be quite easy to change this so that make install works. Also, it would make some of the packaging stuff easier.